File: problemsets/Codeforces/Python/B967.py
"""
 *
 * Author:  Juarez Paulino(coderemite)
 * Email: juarez.paulino@gmail.com
 *
 """
I=lambda:map(int,input().split())
n,a,b=I()
x,*s=I()
s=sorted(s)[::-1]
S=x+sum(s)
for i in range(n):
  if i:S-=s[i-1]
  if x*a>=S*b:break
print(i)

File: problemsets/Codeforces/C++/B967.cpp
/**
 *
 * Author:  Juarez Paulino(coderemite)
 * Email: juarez.paulino@gmail.com
 *
 */
#include <bits/stdc++.h>
using namespace std;
int main() {
  ios::sync_with_stdio(0);
  int n, A, B; cin >> n >> A >> B;
  long long sum = 0;
  int x, s[101010]; cin >> x; sum = x;
  for (int i = 0; i < n-1; i++) cin >> s[i], sum += s[i];
  sort(s, s+n-1, greater<int>());
  int ret = 0;
  for (int i = 0; i < n-1; i++) { 
    if (x*A >= (long long)sum*B) break;
    sum -= s[i]; ret++;
  }
  cout << ret;
}
